Code/Editor/CryEdit.cpp

```diff
@@ -4181,6 +4181,8 @@ extern "C" int AZ_DLL_EXPORT CryEditMain(int argc, char* argv[])
             "\nThis could be because of incorrectly configured components, or missing required gems."
             "\nSee other errors for more details.");
 
+        AzToolsFramework::EditorEventsBus::Broadcast(&AzToolsFramework::EditorEvents::NotifyEditorInitialized);
+
```

Comment:
just thinking out loud, should this go into the block after the "didCryEditStart" flag is checked and before the app->EnableOnIdle() call?

Reply:
or if you want a reliable event, you can skip this idea :)

Reply:
Sure I can move it to there